infra/auth0/main.tf

```diff
@@ -19,9 +19,9 @@ data "auth0_tenant" "tenant" {}
 
 locals {
   custom_domain = {
-    "dev" = "dev.id.online.ntnu.no"
-    "stg" = "stg.id.online.ntnu.no"
-    "prd" = "id.online.ntnu.no"
+    "dev" = "auth.dev.online.ntnu.no"
+    "stg" = "auth.stg.online.ntnu.no"
```

Comment:
Unrelated to this PR, but is it possible to have this be named `auth.staging.online.ntnu.no`?